manage-images.md

```diff
@@ -0,0 +1,120 @@
+An image is the base for a container running in the Anbox Cloud. It contains all necessary components, like Anbox or the Android root file system. Each release of Anbox Cloud comes with an update image, which must be uploaded to the AMS service to make it available to be used inside any container.
+
+AMS can manage different types of images, which can be individually selected by applications. For that, each image must be named when it's initially uploaded to AMS. If the same image receives an update with the next release of Anbox Cloud, it does not need to be re-created but can simply be updated. This will cause all applications using the image to be updated and rebased on the new image version.
+
+Officially released images are available for each version of Anbox Cloud, either on the [Downloads](https://oem-share.canonical.com/partners/indore/share/docs/1.7/en/installation-downloads.html) page (for versions <= 1.7) or on an image server hosted by Canonical.
+
+> **Note**: Anbox images are regular [Ubuntu cloud images](https://cloud-images.ubuntu.com/) where Anbox and its dependencies are installed. Unnecessary packages are removed to improve images size, boot time and security.
+
+## Supported images
+
+The official image server has a set of images available. The following table lists all images and their corresponding Android and Ubuntu versions. Please note the support status for each of the images.
+
+| Name                        | Android Version | Ubuntu Version | Support Status     | Available since |
+|-----------------------------|-----------------|----------------|------------|---------------|
+| `bionic:android11:amd64`      | 11              | 18.04          | supported | 1.10 |
+| `bionic:android11:arm64`      | 11              | 18.04          | supported | 1.10 |
+| `bionic:android10:amd64`      | 10              | 18.04          | supported | 1.0
+| `bionic:android10:arm64`      | 10              | 18.04          | supported | 1.0
+| `bionic:android7:amd64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+| `bionic:android7:arm64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+
+The `bionic:android7:arm64` and `bionic:android7:amd64` images are not receiving Android security updates anymore as Google stopped supporting Android 7 in 2019. They are only provided for legacy reasons and have been dropped with Anbox Cloud 1.10 in April 2021.
```

Comment:
Should these still be listed?

Reply:
Worth checking with @morphis but I feel like we should keep these as long as we mark them unsupported